rskj-core/src/main/java/co/rsk/net/handler/TxTimestamp.java

```diff
@@ -23,6 +23,7 @@
 /**
  * Used to known when a given tx was received by txhandler
  */
+//TODO: Remove it
 class TxTimestamp {
```

Comment:
Since this class isn’t being used, it might be best to remove the TODO and delete the class